docker-compose.yml

```diff
@@ -1,5 +1,40 @@
 version: "3"
 services:
+  celestia-devnet:
+    container_name: celestia_devnet
+    image: ghcr.io/rollkit/local-celestia-devnet:v0.12.1
+    restart: on-failure
+    ports:
+      - "26657"
+      - "26658"
+      - "26659"
+      - "9090"
+
+  evm:
+    container_name: evm_base_shard
+    environment:
+      - DA_AUTH_TOKEN=${DA_AUTH_TOKEN:-} # this value must come from the celestia client.
```

Comment:
Link to this in the comment https://docs.celestia.org/developers/node-tutorial#auth-token